fastapi/routing.py

```diff
@@ -310,7 +310,10 @@ def __init__(
         self.name = get_name(endpoint) if name is None else name
         self.path_regex, self.path_format, self.param_convertors = compile_path(path)
         if methods is None:
-            methods = ["GET"]
+            methods = ["GET", "HEAD"]
+        elif "GET" in methods:
```

Comment:
We can add `HEAD` either here, or on the method. One of those needs to be removed.